### ISSUE TITLE
Visual Interaction Hints to Grab & Drag Balloon 

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#256]
**General Issue** 
Interaction Hints for visual keyboard users are needed for custom interactions or for when the interaction is not implicitly obvious to a user who can see, but who may not use a mouse. A user may use AT such as the keyboard or switch to navigate and interact with the sim. A user who uses a keyboard with a screen reader is able to access structural information and descriptions that guide the interaction. Visual keyboard users, however, may need to first read the Keyboard Shortcuts dialog to gain an understanding of how the sim works with a keyboard., but may have trouble when they just try to use the sim without reading that information.

**Challenge 1: Standard Button Interaction, but Invisible**
The interaction to grab the balloon is executed by pressing the grab balloon button. However, this interaction is not obvious to visual keyboard users because the balloon does not visually resemble a button.

**Challenge 2: Arrow keys & Custom Hot Keys to Drag Balloon**
Once the balloon is grabbed, a user can use the Arrow keys to drag the balloon up, left, down or right. The Arrow keys should be fairly intuitive, but a reassuring hint my be needed. The user can also use a set of custom Hot Keys: W key (to go Up), A key (to go Left), S key (to go Down), or the D key (to go Right). These keys are common to gamers, and with some screen reader software the Arrow keys are unavailable for this interaction.

**People for this Issue**
- TS, MB with help from a visual PhET designer

**Related Docs and Issues**
- @terracoda created pencil sketches
- MB has created some digital sketches

@emily-phet, I am trying out the Milestone function.
I created this issue to put it under the BASE Laundry List Milestones. I think we can create a more specific Milestones once people are assigned and we work out due dates for each issue.

**Comment by terracoda** [comment]
I forgot there was already an active issue related to this topic. See issue #206.

**Comment by terracoda** [comment]
Closing this issue because it is a duplicate of #206.

**Comment by terracoda** [comment]
Oops, I didn't close. This issue is tracked in #206 and #275.
